CHANGELOG/CHANGELOG-0.3.md

```diff
@@ -0,0 +1,13 @@
+## v0.3.0
+
+Changes since `v0.2.1`:
+
+### Features
+
+- Upgrade the `config.kueue.x-k8s.io` API version from `v1alpha1` to `v1alpha2`. `v1alpha1` is no longer supported.
+  `v1alpha2` includes the following changes:
+  - Add Namespace. 
```

Comment:
More details, or it's hard to understand, add namespace for what?

Reply:
Sure, I will add more detailed explanations.

Reply:
Done.